cvat/apps/webhooks/event_type.py

```diff
@@ -0,0 +1,56 @@
+from .models import WebhookTypeChoice
+
+def event_name(action, resource):
+    return f"{action}:{resource}"
+
+class Events:
+    RESOURCES = {
+        "project":      ["create", "update", "delete"],
+        "task":         ["create", "update", "delete"],
+        "issue":        ["create", "delete"], # TO-DO: implement issue_updated
+        "comment":      ["create", "update", "delete"],
+        "invitation":   ["create", "update", "delete"],
+        "membership":   ["update", "delete"],
+        "job":          ["update"],
+        "organization": ["update"]
+    }
+
+    @classmethod
+    def select(cls, resources):
+        ret = set()
+        for resource in resources:
+            ret |= set(f"{event_name(action, resource)}" for action in cls.RESOURCES.get(resource, []))
+        return ret
+
+
+class EventTypeChoice:
+    @classmethod
+    def choices(cls):
+        return sorted((val, val.upper()) for val in AllEvents.events)
+
+
+class AllEvents:
+    webhook_type = "all"
+    events = list(
+        set(
```

Comment:
Why do we need a set here?

Reply:
Just to be confident that we won't have duplicates

Reply:
How is it possible?